baybe/parameters/base.py

```diff
@@ -48,6 +48,10 @@ def is_in_range(self, item: Any) -> bool:
             ``True`` if the item is within the parameter range, ``False`` otherwise.
         """
 
+    @abstractmethod
+    def summary(self) -> dict:
+        """Return a custom summarization of the parameters."""
```

Comment:
Since this method is applied for an individual parameter, change this to `parameter`.

Reply:
Fixed. Thank you!